plugins/inputs/prometheus/metrics_receiver.go

```diff
@@ -48,6 +48,11 @@ type metricAppender struct {
 	batch    PrometheusMetricBatch
 }
 
+func (m *metricAppender) AppendCTZeroSample(storage.SeriesRef, labels.Labels, int64, int64) (storage.SeriesRef, error) {
```

Comment:
Prometheus example does this and we passed our prometheus integ tests so I assume this is fine